stdlib/knative/src/main/ballerina/src/knative/Module.md

```diff
@@ -0,0 +1,36 @@
+## Module Overview
+
+This module offers an annotation based Kubernetes extension implementation for ballerina. 
+
+### Annotation Usage Sample:
+
+```ballerina
+import ballerina/http;
+import ballerina/log;
+import ballerinax/kubernetes;
+
+@kubernetes:Ingress{
+    hostname: "abc.com"
+}
+@kubernetes:Service {
+    name:"hello"
+}
+listener http:Listener helloEP = new(9090);
+
+@kubernetes:Deployment {
```

Comment:
I dont see any knative annotations here.

Reply:
I'll change the .md file for knative